go/libraries/doltcore/sqle/dsess/autoincrement_tracker.go

```diff
@@ -450,10 +470,8 @@ func (a *AutoIncrementTracker) waitForInit() error {
 }
 
 func (a *AutoIncrementTracker) runInitWithRootsAsync(ctx context.Context, roots ...doltdb.Rootish) {
```

Comment:
You have changed the semantics of this method to no longer be async. You've added asyncness back in the one use I see in the diff below, but a) change the name, and b) if there are other uses that expect async behavior, they need to be changed as well